frontend/www/js/omegaup/components/problem/BaseList.vue

```diff
@@ -204,7 +204,9 @@
             </td>
             <td v-else class="text-center align-middle">—</td>
             <td class="text-right align-middle">
-              {{ (100.0 * problem.ratio).toFixed(2) }}%
+              {{ (100.0 * problem.ratio).toFixed(2) }}% <br />({{
```

Comment:
```suggestion
              {{ (100.0 * problem.ratio).toFixed(2) }}%<br />({{
```

ja, el clásico `<br />`